ci-operator/step-registry/single-node/conf/aws/graceful-shutdown/single-node-conf-aws-graceful-shutdown-ref.metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+	"path": "single-node/conf/aws/graceful-shutdown/single-node-conf-aws-graceful-shutdown-ref.yaml",
+	"owners": {
+		"approvers": [
+			"romfreiman",
+			"eranco74",
+			"tsorya",
+			"osherdp",
+			"yuvigold",
+			"omertuc",
+			"wgahnagl",
+			"rphillips",
+			"mrunalp"
+		]
+	}
+}
```

Comment:
can you add a newline here?

Reply:
I believe these are generated files.

Reply:
ah im not familiar.. @wgahnagl is this the case??

Reply:
it is!